# HH_260123 Localization mode enum for supervisor output.
uint8 NORMAL=0
uint8 DEGRADED=1
uint8 DR_ONLY=2
uint8 INVALID=3

uint8 value
string label
